src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String CATEGORY_IN_FILE_SUPPLY = "supply";
+    private static final String CATEGORY_IN_FILE_BUY = "buy";
+    private static final String CATEGORY_IN_FILE_RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeInFile(getResult(readFile(fromFileName)), toFileName);
+    }
+
+    public void writeInFile(String resultInString, String toFileName) {
```

Comment:
why `public` modifier here? please check all other methods modifiers.
also take care about method ordering. 
writing is the last operation according to sequence of executing  and should be ordered in the end of the file.
please also check other method orders.